lib/internal/webidl.js

```diff
@@ -263,6 +264,74 @@ function type(V) {
   }
 }
 
+// https://webidl.spec.whatwg.org/#js-dictionary
+function createDictionaryConverter(members) {
+  // The spec requires us to operate the members of a dictionary in
+  // lexicographical order. We are doing all of these in the outer scope to
+  // reduce the overhead that could happen in the returned function.
+  const sortedMembers = [];
+  for (let i = 0; i < members.length; i++) {
+    const member = members[i];
+    ArrayPrototypePush(sortedMembers, member);
+  }
+
+  ArrayPrototypeSort(sortedMembers, (a, b) => {
```

Comment:
You can do sortedMembers.toSorted() and avoid having the previous for loop and unnecessary operations.

Reply:
Done